File: Assignment No.38/Program4.java
/* Problem statement : Write a program which accept one number from user and Toggle 7 bit and 10 bit of that number if it is on.return modified number.
Input : 137
Output : 713
*/
import java.util.*;

class Program4
{
	public static int ToggleBit(int iNo)
	{
		int iResult=0;
		int iMask=0x00000240;

		return (iNo ^ iMask);
		
		
	}
	public static void main(String args[])
	{
		Scanner sobj=new Scanner(System.in);
		int iNo=0;
		System.out.println("Enter the Number :");
		iNo=sobj.nextInt();

		int iRet=0;

		iRet=ToggleBit(iNo);
		System.out.println("Modified Number :"+iRet);
	}
}

File: Assignment No 4/Program4.c
//Write a Program which accept number from user and return summation of all its non factor.
//Input : 12
//Output : 50
//Input : 10
//Output : 37
#include<stdio.h>

int SumNonFact(int iNo)
{
	int iFact=0;
	int iSum=0;

	for(iFact=1;iFact<=iNo;iFact++)
	{
		if((iNo%iFact)!=0)
		{
			iSum=iSum+iFact;
		}
	}
	return iSum;
}
int main()
{
	int iValue =0;
	int iRet=0;

	printf("Enter the number :\n");
	scanf("%d",&iValue);

	iRet=SumNonFact(iValue);
	printf("Summation of non factors are :%d",iRet);
	
	return 0;
}